lib/components/button/button.dart

```diff
@@ -12,6 +12,7 @@ enum ButtonType {
   add,
   remove,
   confirm,
+  cancel,
   edit,
```

Comment:
nit: can we change the order to:
```suggestion
  add,
  remove,
  edit,
  confirm,
  cancel,
```
and also adjust the switch below by button type to use this same order?